src/components/GetStartedCard/GetStartedCard.tsx

```diff
@@ -64,15 +71,50 @@ const GetStartedCard = () => {
                   browser.
                 </ListItem>
               </List>
+              {error ? (
+                <Alert
+                  title="An error occurred"
+                  variant={AlertVariant.danger}
+                  actionClose={<AlertActionCloseButton onClose={() => setError(undefined)} />}
+                  isInline
+                  className="pf-u-mb-lg"
+                >
+                  {error}
+                </Alert>
+              ) : null}
               <AnalyticsButton
-                isDisabled={state.status === 'pending-approval'}
-                onClick={() => setShowUserSignup(true)}
+                isDisabled={
+                  loading || state.status === 'pending-approval' || state.status === 'provisioning'
+                }
+                isLoading={loading || state.status === 'provisioning'}
+                onClick={async () => {
+                  if (state.status === 'new') {
+                    (async () => {
+                      try {
+                        setError(undefined);
+                        setLoading(true);
+                        await signup();
+                        await refreshSignupData();
+                      } catch (e) {
+                        setError(errorMessage(e));
+                      } finally {
+                        setLoading(false);
+                      }
+                    })();
```

Comment:
I don't think we have to wrap this into IIFE and `async` the onClick is already marked as `async`.